lib/install.js

```diff
@@ -105,8 +105,9 @@ const fs = require('fs'),
                     next();
                 },
                 function(next) {
+                    console.log("install__")
                     const md5 = crypto.createHash('md5'),
-                        buffer=new Buffer(md5DataSize);
+                        buffer=Buffer.alloc(md5DataSize);////
```

Comment:
1. Please remove "////"
2. Add a space between = and Buffer
butter = Butter.alloc(..)